docs/source/60_building_apps.md

```diff
@@ -256,6 +256,39 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s ShuffleRegionJoin API
```

Comment:
Ah sorry, sed ;). I was saying to delete "shuffle"; this section should be docs for all region join implementations.